src/orion/algo/mofa/_version.py

```diff
@@ -0,0 +1,677 @@
+# This file helps to compute a version number in source trees obtained from
```

Comment:
This file should be removed. That was only relevant for the external plugin, not as part of the core repo.

Reply:
Addressed in https://github.com/Epistimio/orion/pull/864/commits/268378e0e47b56f5d9fe35f35fa77a1cb2a99aae